pkg/manager/manager.go

```diff
@@ -206,6 +205,13 @@ func (m MediaManager) AddMovieToLibrary(ctx context.Context, request AddMovieReq
 		return nil, err
 	}
 
+	dcs, err := m.ListDownloadClients(ctx)
```

Comment:
I think we can remove the download client id from the add movie request now

Reply:
Should be gone... [line 194](https://github.com/kasuboski/mediaz/pull/18/files#diff-d500fbd6a2aa620607ca5e2a7c3ac4f1a4c82309d1a549561e92abfcb18f2f0eL194)